### DISCUSSION TITLE
Syndicate surcharges

### DISCUSSION THREAD

**Discussion by movingpictures** [endless-sky/endless-sky#8246] [Ideas]
The Syndicate likes making money.
If the Syndicate could, it would put tolls on hyperspace lanes, but since those are public domain (as it were) the next best thing is landing fees for starports; or charges for fuel at starports.

**Comment by DoublePants** [comment]
But then they'd make less money, because they'd be disincentivizing ships from landing in their space. If you're a trader and have the whole of space available to you and in one region you can sell your goods without paying a docking fee and in one region you can't, why would you ever go to the region you can't?

If we want to depict them as a capitalist dystopia, then if anything they'd be subsidizing businessmen who come to their space at the expense of extracting resources from the people who are too poor to flee their planets, since the dystopian version of capitalism is to take from the poor and give to the rich. As a starship captain, even one who only owns a little shuttle, you're the rich.
